### PR TITLE
Add cost_centre parameter to all environments

### PR DESCRIPTION
**Pull request by suzannehamilton**
Add a parameter store variable to all environments, so that the environment-specific Terraform scripts can use it to set the `CostCentre` tag on other resources.

Also update the permissions so that the parameter can be read by the environment-specific Terraform scripts.

We might eventually replace this with a private git repo containing global varibles, but this solution is easier to set up so let's try it for now.

@TomJKing @MancunianSam Does this fit into the way you're doing things in Terraform? I've created a new policy because the parameter read permissions didn't seem to fit into any existing ones - so you have any concerns about hitting IAM limits?

Edit: ~Or should I have put the new permissions into shared_terraform_policy? I think I misunderstood what that was for.~ I've moved the SSM permissions into shared_terraform_policy.

### CODE REVIEW

File: root.tf [hunk 1]
```diff
@@ -107,6 +107,40 @@ module "prod_environment_role" {
   tdr_mgmt_account_number = data.aws_ssm_parameter.mgmt_account_number.value
 }
 
+//Shared parameters to store in each environment
+module "intg_environment_paramers" {
```
Comment (TomJKing) [inline root.tf]: Could put the module name out in full, ie: intg_environment_parameters

Comment (suzannehamilton) [reply root.tf]: Oops, that's a typo. Fixed!

**Comment by suzannehamilton** [comment]
@TomJKing When is TDRJenkinsReadParams<env> used rather than TDRTerraformRole<env>? Is the former used by deployments?

**Comment by suzannehamilton** [comment]
@TomJKing The other thing I wanted to check was the parameter namespacing.

Should the environment-specific parameters be something like `/intg/cost_centre` because it's being stored in the environment-specific account, or something like `/mgmt/cost_centre` because it's a common parameter that was created by the backend Terraform?

**Comment by suzannehamilton** [comment]
@TomJKing I've moved the parameters to /mgmt.

**Pull request merged**